### DISCUSSION TITLE
Questions Regarding Error #3691

### DISCUSSION THREAD

**Discussion by cheezboi** [BowlerHatLLC/vscode-as3mxml#547] [Q&A]
To all who may know: I've been using VSC with the AS3MXML extension to update some older AS3 projects. These are for a desktop/stand alone application, so they use lots of large graphics. I've run into the "Error #3691: Resource limit for this resource type exceeded." issue and I am not sure what I can do to try and solve the issue. 

(I've been commenting out .atf files in my asset manager until I can get the project to load, but then it will, of course, fail when it is missing an asset.)

I was told that if I wanted to use lots of graphic files, I could: "Ensure 64bit in the application.xml" by adding this entry: "`<architecture>64</architecture>`" -  but I do not see anything like that in VSC. It was also suggested to make sure the "build target is "bundle" - which I believe is the default setting. 

This could clearly be me not understanding the asset manager correctly or that I am trying to load too many .atf files. 

My questions would be for VSC AS3MXML:
- 1) is there a 64 bit setting?
- 2) is there a 'bundle' setting?
- 3) is there a way to know how much enqueued and loaded xml/atf files a project can have?

Any advice is appreciated and welcome! Thank you - MK

**Comment by joshtynjala** [comment]
> Ensure 64bit in the application.xml" by adding this entry: "`<architecture>64</architecture>`" - but I do not see anything like that in VSC

i. This would go into the _-app.xml_ for your Adobe AIR app. Nothing specific to VSCode here.

> It was also suggested to make sure the "build target is "bundle" - which I believe is the default setting.

ii. The task named **ActionScript: package Windows release (captive runtime) - asconfig.json** (or macOS instead of Windows) will produce a captive runtime "bundle".

In the `airOptions` section of _asconfig.json_, you can force this type of build by setting the `"target"` field to `"bundle"`, but this task should be available by default without needing to set that value.

> is there a way to know how much enqueued and loaded xml/atf files a project can have?

iii. This is more a question related to Adobe AIR or Starling. You should probably ask somewhere else about this.

**Reply by cheezboi** [reply]
Thanks for the reply Josh - I do not seem to have a "-app.xml", probably because this is a standard .swf project and not and AIR app. (just guessing!) And from what I've read so far, it is defaulted to "bundle". Again, thanks for the reply - I will continue to look for a solution, but it might not use VSC, unfortunately.
